_events/sr2024/virtual-kickstart.md

```diff
@@ -16,12 +16,17 @@ streaming the event we strongly encourage teams to meet up and watch the stream
 together. This is a great way to get to know your team mates and start working
 together on the kit.
 
+Throughout the afternoon we'll have Blueshirts (our awesome volunteers!) in the Discord server, ready to answer any questions you may have and to help you through the microgames.
```

Comment:
suggestion: maybe link to the Discord page in the docs?

Reply:
👍 [6adef6b](https://github.com/srobo/website/pull/525/commits/6adef6b85d09ad104fcfc69cbcd7aadd669cd90c)